keyboards/owlab/suit80/ansi/info.json

```diff
@@ -0,0 +1,100 @@
+{
+    "keyboard_name": "suit80",
+    "url": "https://www.qwertykeys.com/products/jrrgvdsyu47wer",
+    "maintainer": "Owlab",
+    "width": 18.25,
+    "height": 6.25,
```

Comment:
```suggestion
```
The `width` and `height` fields in `info.json` are no longer needed — the configurator code can calculate the dimensions from the layout.